File: translation_output/mistral/codenet/java/python/p03777.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```:
```
import java.util.*;

public class p03777{
  public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    String a = sc.next();
    String b = sc.next();
    if(a.equals(b)){
      System.out.println("H");
    }else{
      System.out.println("D");
    }
  }
}

```
[/INST]
```python
import java.util

def main():
    sc = java.util.Scanner(input)
    a = sc.next()
    b = sc.next()
    if a == b:
        print("H")
    else:
        print("D")

main()
```</s>

File: Results/DER/Translation/translation_output/misleading_test/gpt-4/codenet/python/java-sanitized/p03777.java
import java.util.Scanner;

public class p03777 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String a = scanner.next();
        String b = scanner.next();

        if (a.equals("H")) {
            System.out.print(b);
        } else {
            if (b.equals("H")) {
                System.out.print("D");
            } else {
                System.out.print("H");
            }
        }
    }
}